tt-media-server/tt_model_runners/vllm_forge_qwen_embedding_runner.py

```diff
@@ -0,0 +1,77 @@
+
+from config import settings
+from config.settings import get_settings
+from domain.text_completion_request import TextCompletionRequest
+from domain.text_embedding_request import TextEmbeddingRequest
+from tt_model_runners.base_device_runner import BaseDeviceRunner
+from transformers import AutoTokenizer
+from utils.helpers import log_execution_time
+from utils.logger import TTLogger
+import vllm
+
+
+class VLLMForgeEmbeddingQwenRunner(BaseDeviceRunner):
+
+    def __init__(self, device_id: str):
+        super().__init__(device_id)
+        self.settings = get_settings()
+        self.pipeline = None
+        self.logger = TTLogger()
+
+    def get_device(self):
+        return None
+
+    def close_device(self, device) -> bool:
+        return True
+
+    @log_execution_time("Model warmup")
+    async def load_model(self, device)->bool:
+        self.logger.info(f"Device {self.device_id}: Loading model...")
+
+        self.tokenizer = AutoTokenizer.from_pretrained(settings.SupportedModels.QWEN_3_EMBEDDING_4B.value)
```

Comment:
For readability and to avoid confusion, I suggest importing `SupportedModels` directly:
`from config.constants import SupportedModels`. Then you can use `SupportedModels.QWEN_3_EMBEDDING_4B.value` instead of `settings.SupportedModels.QWEN_3_EMBEDDING_4B.value`.
The same applies to line 37.

```suggestion
        self.tokenizer = AutoTokenizer.from_pretrained(SupportedModels.QWEN_3_EMBEDDING_4B.value)
```